src/components/client/CommercesList.js

```diff
@@ -47,15 +44,21 @@ class CommercesList extends Component {
     });
 
     this.props.onFavoriteCommercesRead();
+    if (this.props.provinceNameFilter == null) {
```

Comment:
Esto es lo mismo que hacer `!this.props.provinceNameFilter` (no es ni triple igualdad `===`). Quedaria mas compacto asi

Reply:
pasa que al principio tenia que hacer diferencia entre `null` y `''` , pero después que me dijiste que cuando se volvía atrás, los filtros no tenían que ser persistentes, ya no haría falta esa diferenciación. Faltó cambiar eso.